lightning_transformers/core/callback.py

```diff
@@ -11,11 +11,123 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+import collections
+import inspect
+import os
 import time
+from typing import Any, Dict, List, Optional, Union
 
+import numpy
+import onnxruntime
 import torch
+from pl_bolts.callbacks import SparseMLCallback
 from pytorch_lightning import Callback
+from pytorch_lightning.loggers import WandbLogger
 from pytorch_lightning.utilities import rank_zero_info
+from pytorch_lightning.utilities.exceptions import MisconfigurationException
+from sparseml.pytorch.utils import ModuleExporter
+from sparseml.pytorch.utils.logger import WANDBLogger
+from torch import Tensor
+
+
+class LightningBoltsSparseMLCallback(SparseMLCallback):
```

Comment:
```suggestion
class TranformerSparseMLCallback(SparseMLCallback):
```

Reply:
👍 I'll commit that with the correct spelling of Transformer